lstchain/tools/lstchain_create_time_calibration_file.py

```diff
@@ -0,0 +1,97 @@
+"""
+Create drs4 time correction coefficients.
+"""
+import glob
+
+from ctapipe.core import Provenance, Tool, traits
+from ctapipe.io import EventSource
+from lstchain.calib.camera.time_correction_calculate import TimeCorrectionCalculate
+from tqdm.autonotebook import tqdm
+
+
+class TimeCalibrationHDF5Writer(Tool):
+    name = "TimeCalibrationHDF5Writer"
+    description = "Generate a HDF5 file with time calibration coefficients"
+
+    input = traits.Path(
+        help="Path to the fits.fz events file or directory to glob"
+    ).tag(config=True)
+
+    glob = traits.Unicode(
+        help="Filename pattern to glob files in the directory",
+        default_value="*"
+    ).tag(config=True)
+
+    progress_bar = traits.Bool(
+        help="show progress bar during processing",
+        default_value=True,
+    ).tag(config=True)
+
+    aliases = {
+        ("i", "input"): "TimeCalibrationHDF5Writer.input",
+        ("o", "output"): "TimeCorrectionCalculate.calib_file_path",
+        "glob": "TimeCalibrationHDF5Writer.glob",
+        "max-events": "EventSource.max_events",
+        "pedestal": "LSTR0Corrections.drs4_pedestal_path",
+        "dragon-reference-time": "EventTimeCalculator.dragon_reference_time",
+        "dragon-reference-counter": "EventTimeCalculator.dragon_reference_counter",
```

Comment:
Also add `module_id` since you need all three now. And with the next release, there will also be the option to just give the path to the run summary instead of giving all the counters explicitly.